understanding/non-text-content.html

```diff
@@ -601,23 +452,17 @@ <h2>Related Resources</h2>
                </ul>
             </section>
             <section id="techniques">
-               <h2>Techniques</h2>
-               <p>Each numbered item in this section represents a technique or combination of techniques
-                  that the WCAG Working Group deems sufficient for meeting this Success Criterion. However,
-                  it is not necessary to use these particular techniques. For information on using other
-                  techniques, see <a href="understanding-techniques">Understanding Techniques for WCAG Success Criteria</a>, particularly the "Other Techniques" section.
+               <h2>達成方法</h2>
```

Comment:
ここは、「テクニック」に差し替えてもらえればと
たとえば：https://waic.jp/translations/WCAG22/Understanding/dragging-movements.html#techniques のように（以下、達成方法→テクニックととりあえず置換してもらえればと）。

```suggestion
               <h2>達成方法</h2>
```